flink-tests/src/test/java/org/apache/flink/test/example/failing/JobSubmissionFailsITCase.java

```diff
@@ -80,19 +75,6 @@ private static JobGraph getWorkingJobGraph() {
 		return new JobGraph("Working testing job", jobVertex);
 	}
 
-	// --------------------------------------------------------------------------------------------
-
-	private final boolean detached;
-
-	public JobSubmissionFailsITCase(boolean detached) {
-		this.detached = detached;
-	}
-
-	@Parameterized.Parameters(name = "Detached mode = {0}")
-	public static Collection<Boolean[]> executionModes(){
-		return Arrays.asList(new Boolean[]{false},
-				new Boolean[]{true});
-	}
```

Comment:
Why can we remove the detached test case here? Aren't we losing test coverage by removing this code?

Reply:
We can not distinguish between detached and attached job submission anymore. All submissions are detached.

This is the old code:
```java
			if (detached) {
				client.submitJob(jobGraph).get();
			} else {
				submitJobAndWaitForResult(client, jobGraph, getClass().getClassLoader());
			}
// method for reference:
	public static void submitJobAndWaitForResult(ClusterClient<?> client, JobGraph jobGraph, ClassLoader classLoader) throws Exception {
		client.submitJob(jobGraph)
			.thenCompose(client::requestJobResult)
			.get()
			.toJobExecutionResult(classLoader);
	}
```
... and with the non-blocking job submission, we would basically do the same in both cases.

Reply:
Thanks for the explanation.